docs/source/workflow_inputs.rst

```diff
@@ -1052,6 +1066,26 @@ If a room air conditioner is specified, additional information is entered in ``C
 
   .. [#] If AnnualCoolingEfficiency[Units="EER" or Units="CEER"]/Value not provided, defaults to EER from the lookup table that can be found at ``HPXMLtoOpenStudio\resources\lu_hvac_equipment_efficiency.csv`` based on YearInstalled.
 
+.. _hvac_cooling_ptac:
+
+Packaged Terminal Air Conditioner
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If a ptac is specified, additional information is entered in ``CoolingSystem``.
+
+  ===================================================================  =================  ===========  ===============  ========  =========  ==================================
+  Element                                                              Type               Units        Constraints      Required  Default    Notes
+  ===================================================================  =================  ===========  ===============  ========  =========  ==================================
+  ``YearInstalled`` or ``AnnualCoolingEfficiency[Units="EER"]/Value``  integer or double  # or Btu/Wh  > 1600 or > 0    Yes       See [#]_   Year installed or Rated efficiency
+  ``CoolingCapacity``                                                  double             Btu/hr       >= 0             No        autosized  Cooling capacity
+  ``SensibleHeatFraction``                                             double             frac         0 - 1            No                   Sensible heat fraction
+  ``CompressorType``                                                   string             See [#]_                      No        See [#]_   Type of compressor
+  ===================================================================  =================  ===========  ===============  ========  =========  ==================================
+
+  .. [#] If AnnualCoolingEfficiency[Units="EER"]/Value not provided, defaults to EER from the lookup table that can be found at ``HPXMLtoOpenStudio\resources\lu_hvac_equipment_efficiency.csv`` based on YearInstalled.
+  .. [#] The only CompressorType choice supported is "single stage" for now.
+  .. [#] If CompressorType not provided, defaults to "single stage".
```

Comment:
Remove references to `CompressorType`. Like room ACs, it's not an input called out in EPvalidator.xml.